lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -421,28 +421,29 @@ switchport_mode_private_vlan_host_promiscous:
 
 
 switchport_mode_private_vlan_trunk_promiscuous:
-  _exclude: [ios_xr, N8k]
+  _exclude: [ios_xr, N3k, N8k]
```

Comment:
Does I4 (Dublin+ support this? Why does N9k support this, but not N3k?  They are running the same image.